GameData/FerramAerospaceResearch/FARConfig.cfg

```diff
@@ -392,4 +392,289 @@ FARConfig
+            color = 0.705882, 0.003922, 0.368627
+            color = 0.254902, 0.000000, 0.274510
+            color = 0.615686, 1.000000, 0.811765
+            color = 0.894118, 0.282353, 0.615686
+            color = 0.890196, 0.890196, 0.278431
+            color = 0.862745, 0.886275, 0.647059
+            color = 0.000000, 0.156863, 0.352941
+            color = 0.666667, 0.356863, 0.509804
+            color = 0.000000, 0.000000, 0.862745
+            color = 0.294118, 0.305882, 0.317647
+            color = 0.854902, 0.749020, 0.835294
+            color = 0.000000, 0.301961, 0.600000
+            color = 0.533333, 0.392157, 0.619608
+            color = 0.415686, 0.117647, 0.113725
+            color = 0.556863, 0.321569, 0.772549
+            color = 0.721569, 0.854902, 0.874510
+            color = 0.866667, 0.701961, 0.992157
+            color = 0.482353, 0.282353, 0.329412
+            color = 0.298039, 0.450980, 0.000000
+            color = 0.270588, 0.000000, 0.466667
+            color = 0.698039, 0.372549, 0.000000
+            color = 0.572549, 0.819608, 0.525490
+            color = 0.333333, 0.200000, 0.298039
+            color = 0.411765, 0.690196, 0.521569
+            color = 0.670588, 0.576471, 0.690196
+            color = 0.905882, 0.329412, 0.258824
+            color = 0.560784, 0.549020, 0.541176
+            color = 0.439216, 0.678431, 0.317647
+            color = 0.670588, 0.486275, 0.454902
+            color = 0.000000, 0.203922, 0.235294
+            color = 0.145098, 0.058824, 0.074510
+            color = 0.905882, 0.690196, 0.000000
+            color = 0.478431, 0.800000, 0.862745
+            color = 0.094118, 0.078431, 0.227451
+            color = 0.615686, 0.321569, 0.223529
+            color = 0.733333, 0.482353, 0.192157
+            color = 0.717647, 0.792157, 0.580392
+            color = 0.192157, 0.031373, 0.000000
+            color = 0.639216, 0.584314, 0.023529
+            color = 0.000000, 0.854902, 0.729412
+            color = 0.454902, 0.627451, 0.870588
+            color = 0.388235, 0.235294, 0.450980
+            color = 1.000000, 0.854902, 0.560784
+            color = 0.466667, 0.721569, 0.000000
+            color = 0.250980, 0.184314, 0.113725
+            color = 0.345098, 0.529412, 0.349020
+            color = 0.176471, 0.000000, 0.129412
+            color = 0.960784, 0.631373, 0.831373
+            color = 0.854902, 0.000000, 0.666667
+            color = 0.462745, 0.160784, 0.286275
+            color = 0.741176, 0.898039, 0.000000
+            color = 0.764706, 0.760784, 0.364706
+        }
+    }
+
+    Occlusion {
+        useRaycaster = false
+        jobsPerThread = 3
+        maxJobs = 10
+        // 1000 points produces typical misses 2-3 degrees when optimized for average miss distance.
+        fibonacciLatticeSize = 2000
+        maxRaycastDimension = 100
+        raycastResolution = 0.1
+        resetInterval = 180
+        velocityThreshold = 0.01
+    }
```

Comment:
Comments on what each of the options do would be welcome

Reply:
Fixed in a737ed8